ggml/src/ggml-cuda/ggml-cuda.cu

```diff
@@ -261,6 +261,9 @@ static ggml_cuda_device_info ggml_cuda_init() {
         GGML_LOG_INFO("  Device %d: %s, %s (0x%x), VMM: %s, Wave Size: %d\n",
                       id, prop.name, prop.gcnArchName, info.devices[id].cc & 0xffff,
                       device_vmm ? "yes" : "no", prop.warpSize);
+#elif defined(GGML_USE_MUSA)
+        info.devices[id].smpbo = prop.sharedMemPerBlockOptin;
+        info.devices[id].cc = 100*prop.major + 10*prop.minor;
```

Comment:
Why is there a separate branch for MUSA here?

Reply:
This is for better adapting MUSA cc in the future, the current `cc` calculation can not reflect the actual CC level of MUSA.